internal/consensus/state.go

```diff
@@ -255,6 +255,12 @@ func (cs *State) GetRoundState() *cstypes.RoundState {
 	return &rs
 }
 
+// getRoundState returns a shallow copy of the internal consensus state.
```

Comment:
Having both `GetRoundState` and `getRoundState` with the same description is confusing. Can we add a comment to `GetRoundState` about why we need an RLock there and to `getRoundState` why we don't need an RLock there?

Reply:
Because `GetRoundState` also gets shallow copy, but with the lock.

Reply:
Oh ok, can do. I thought the comet convention was that capitalized `Get`'s take the RLock, and lowercase don't

Reply:
A solution here is to have a private `getRoundState()` method, with a comment saying that it is not thread safe, and a private `GetRoundState()` that acquires the lock and calls the former.